lib/elixir/lib/io.ex

```diff
@@ -32,6 +29,109 @@ defmodule IO do
   was last accessed. The position of files can be changed using the
   `:file.position/2` function.
 
+  ## Iolists and iodata
+
+  Iolists and iodata are data structures that can be used as a sometimes
+  more efficient alternative to binaries.
```

Comment:
"that can be used as a more efficient alternative to binaries under certain situations".